tree.go

```diff
@@ -1,9 +1,9 @@
 package rsmt2d
 
-import (
-	"crypto/sha256"
+import "fmt"
 
-	"github.com/celestiaorg/merkletree"
+const (
+	Default = "Default-tree"
```

Comment:
[nit] can this constant clarify what it is a default for? Additionally can we use consistent casing for the string? Pheraps:

```suggestion
	DefaultTree = "default-tree"
```

Reply:
[question] should this constant exist in `default_tree.go` instead of in this file?

Reply:
hmmm make sense